### DISCUSSION TITLE
How do you guys feel with the xxx (class 3) enemy vessels?

### DISCUSSION THREAD

**Discussion by Neuron114** [PavelZinchenko/event-horizon-main#471] [General]
Hello community of Event Horizon
Ever since this feature being added, it's one hell of a controversy for the Event Horizon community until now. I wonder how anyone feels about this feature

Personally, despite being able to adapt and finally manage to build a vessel that can beat each of them, the pushing to get better feels frustrating along the way. And knowing that the capital xxx vessels aren't even implemented due to how difficult they can be, I knew that this doesn't feels right. Yes I did feels quite superior able to beat them 100 levels higher than me but this still bothers me that they don't give meaningful reward, at least for now in Event Horizon: space rpg

**Comment by TaipionsSalemClient** [comment]
I feel the overly excessive grind to get anything in EH, plus the incredibly shitty "crafting system" make it about 10x harder and 100x as annoying as it should be, to get the right tools to deal with xxx ships.
Plus some modules that are a hard requirement to make certain ships work are even more, far far far more bullshit to get.
Like: A good Scarab build relies on reactive armour, good luck getting that in gold quality in EH.

Adding a proper crafting system and reducing the required grind slightly, plus allowing access to scavenger tech other than random merchant encounters would balance xxx ships far better than just nerfing them.
